cfme/tests/services/test_catalog_item.py

```diff
@@ -92,11 +95,6 @@ def test_update_catalog_item(catalog_item):
         catalog_item.description = "my edited item description"
 
 
-def test_delete_catalog_item(catalog_item):
```

Comment:
Removed this test because the yield fixture catalog item also has a clean up delete . 
This test delete catalog item and then tries to delete it again as clean up .